src/wp-admin/includes/class-wp-list-table.php

```diff
@@ -388,7 +388,15 @@ public function search_box( $text, $input_id ) {
 		$input_id = $input_id . '-search-input';
 
 		if ( ! empty( $_REQUEST['orderby'] ) ) {
-			echo '<input type="hidden" name="orderby" value="' . esc_attr( $_REQUEST['orderby'] ) . '" />';
+			if ( is_array( $_REQUEST['orderby'] ) ) {
+				foreach ( $_REQUEST['orderby'] as $key => $value ) {
+					// Because arrays in query parameters are not standardized, the orderby parameter can be an associative or a non-associative array.
+					$key = is_int( $key ) ? '' : $key;
```

Comment:
```suggestion
```

This isn't required as PHP will handle `orderby[0]`, etc.

Reply:
You're right! 

But I think we at least need to make sure we're passing a `string` to the `esc_attr( $key )`, to make sure we comply with the function's signature.

So, I made that into a simple cast to string (which keeps the functionality intact, comparing it to what you proposed 👍)